config/prometheus/monitor.yaml

```diff
@@ -1,16 +1,18 @@
-
 # Prometheus Monitor Service (Metrics)
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   labels:
     control-plane: controller-manager
-  name: controller-manager-metrics-monitor
+  name: capsule-monitor
   namespace: system
 spec:
   endpoints:
-    - path: /metrics
+    - interval: 15s
+      path: /metrics
       port: metrics
+  jobLabel: app.kubernetes.io/name
```

Comment:
According to the Prometheus Operator APIs:

```
job label: The label to use to retrieve the job name from
```

Since this is the installation with `kustomize`, we have to use the value `controller-manager`.